fe/fe-core/src/main/java/com/starrocks/alter/LakeRollupJob.java

```diff
@@ -193,8 +194,13 @@ protected void runPendingJob() throws AlterCancelException {
                 Map<Long, Long> tabletIdMap = this.physicalPartitionIdToBaseRollupTabletIdMap.get(partitionId);
                 for (Tablet rollupTablet : rollupIndex.getTablets()) {
                     long rollupTabletId = rollupTablet.getId();
-                    ComputeNode computeNode = GlobalStateMgr.getCurrentState().getWarehouseMgr()
-                            .getComputeNodeAssignedToTablet(warehouseId, (LakeTablet) rollupTablet);
+                    ComputeNode computeNode = null;
+                    try {
+                        computeNode = GlobalStateMgr.getCurrentState().getWarehouseMgr()
+                                .getComputeNodeAssignedToTablet(warehouseId, (LakeTablet) rollupTablet);
+                    } catch (ErrorReportException e) {
+                        // computeNode is null
+                    }
                     if (computeNode == null) {
```

Comment:
Can this move to catch block?

Reply:
no, `getComputeNodeAssignedToTablet` may return null